ace/scrape.py

```diff
@@ -93,6 +94,26 @@ def elink(self, pmid, retmode='ref', **kwargs):
         }
         response = self.get("elink", params=params, **kwargs)
         return response
+    
+    def elink_PMC(self, pmid, **kwargs):
```

Comment:
Maybe we can refactor `elink` to take "db" as a parameter, and based on that query either pubmed or pubmed central.

We could do this for `efetch` as well.

The default parameter could be `pubmeb` which would not break existing code, but then it would mean we don't need two extra functions.